src/common/file-upload/file-upload.html

```diff
@@ -0,0 +1,12 @@
+<div class="form-control input-auto input-large fileinput-exists" data-trigger="fileinput">
+    <span class="glyphicon glyphicon-file"></span>&nbsp;
+    <span class="fileinput-preview"></span>
+</div>
+<span class="btn btn-default btn-file">
+        <span class="fileinput-new">{{msg('server.button.selectFile')}}</span>
+        <span class="fileinput-exists">{{msg('server.button.changeFile')}}</span>
+        <input type="file" id="fileInput" name="file" class="input-file"/>
+    </span>
+<a href="#" class="btn btn-danger fileinput-exists" data-dismiss="fileinput">
+    <i class="fa fa-fw fa-trash-o"></i>&nbsp;{{msg('server.button.remove')}}
+</a>
```

Comment:
Couldn't it be a `<button></button>` element with a class `button-delete` from [common/base](https://github.com/motech/motech-ui/tree/master/src/common/base)?